src/pmdas/farm/help

```diff
@@ -0,0 +1,705 @@
+@farm.ata.error_statistics.index_of_last_flash_led Index of the last Flash LED for the drive
+@farm.ata.error_statistics.flash_led_event_0.event_information Flash LED Event 0: Event information
+@farm.ata.error_statistics.flash_led_event_0.timestamp_of_event Flash LED Event 0: Timestamp of Event
+@farm.ata.error_statistics.flash_led_event_0.power_cycle_event Flash LED Event 0: Power Cycle Event
+@farm.ata.error_statistics.flash_led_event_1.event_information Flash LED Event 1: Event information
+@farm.ata.error_statistics.flash_led_event_1.timestamp_of_event Flash LED Event 1: Power Cycle Event
+@farm.ata.error_statistics.flash_led_event_1.power_cycle_event Flash LED Event 1: Power Cycle Event
+@farm.ata.error_statistics.flash_led_event_2.event_information Flash LED Event 2: Event information
+@farm.ata.error_statistics.flash_led_event_2.timestamp_of_event Flash LED Event 2: Power Cycle Event
+@farm.ata.error_statistics.flash_led_event_2.power_cycle_event Flash LED Event 2: Power Cycle Event
+@farm.ata.error_statistics.flash_led_event_3.event_information Flash LED Event 3: Event information
+@farm.ata.error_statistics.flash_led_event_3.timestamp_of_event Flash LED Event 3: Power Cycle Event
+@farm.ata.error_statistics.flash_led_event_3.power_cycle_event Flash LED Event 3: Power Cycle Event
+@farm.ata.error_statistics.flash_led_event_4.event_information Flash LED Event 4: Event information
+@farm.ata.error_statistics.flash_led_event_4.timestamp_of_event Flash LED Event 4: Power Cycle Event
+@farm.ata.error_statistics.flash_led_event_4.power_cycle_event Flash LED Event 4: Power Cycle Event
+@farm.ata.error_statistics.flash_led_event_5.event_information Flash LED Event 5: Event information
+@farm.ata.error_statistics.flash_led_event_5.timestamp_of_event Flash LED Event 5: Power Cycle Event
+@farm.ata.error_statistics.flash_led_event_5.power_cycle_event Flash LED Event 5: Power Cycle Event
+@farm.ata.error_statistics.flash_led_event_6.event_information Flash LED Event 6: Event information
+@farm.ata.error_statistics.flash_led_event_6.timestamp_of_event Flash LED Event 6: Power Cycle Event
+@farm.ata.error_statistics.flash_led_event_6.power_cycle_event Flash LED Event 6: Power Cycle Event
+@farm.ata.error_statistics.flash_led_event_7.event_information Flash LED Event 7: Event information
+@farm.ata.error_statistics.flash_led_event_7.timestamp_of_event Flash LED Event 7: Power Cycle Event
+@farm.ata.error_statistics.flash_led_event_7.power_cycle_event Flash LED Event 7: Power Cycle Event
+@farm.ata.error_statistics.uncorrectable_errors Uncorrectable errors count 
+Number of uncorrectable errors for the drive
+
+@farm.ata.error_statistics.cumulative_lifetime_unrecoverable_errors_due_to_erc Cumulative unrecoverable read errors ERC
+Number of cumulative lifetime unrecoverable read errors due to ERC
+
+@farm.ata.error_statistics.cum_lifetime_unrecoverable_errors_head_0.cumulative_lifetime_unrecoverable_read_repeating Head 0
```

Comment:
Similarly, this and related metrics seems to me like it should have a 'head' instance domain - fewer metrics this way - its more efficient for client tools, more flexible, easier to maintain longer term.